### ISSUE TITLE
fix docs for transition

### ISSUE THREAD

**Issue by joamatab** [gdsfactory/gdsfactory#2131]
The notebook has some issues since last release that I can't fully understand

https://gdsfactory.github.io/gdsfactory/notebooks/rib_strip_autotransition.html

@tvt173

**Comment by tvt173** [comment]
yeah, and what are all these warnings? it would be good if there is at least a way to suppress them for users who routinely have non-90 ports

**Comment by joamatab** [comment]
https://github.com/gdsfactory/gdsfactory/pull/2135

**Comment by discdiver** [comment]
Can this issue be closed or transitioned to another issue? The notebook linked above now 404s.

**Comment by joamatab** [comment]
Closing as superseded by the current documentation. The linked `rib_strip_autotransition` notebook was removed, while `docs/notebooks/03_Path_CrossSection.ipynb` now documents and executes cross-section transitions, arbitrary-path transitions, asymmetric transitions, section-name matching, and `skip_transition`. The current routing notebook also documents automatic layer/width transitions.
